spec/models/upload_concepts.rb

```diff
@@ -57,7 +57,14 @@
             end
         end
     end
-    
+    describe 'import corrupt concepts.csv with missing fields' do
+        it 'lets you know if concept name is missing' do
+            expect(Concept.verify_row(nil, "")).to eq "Concept name for one of the concepts is missing. Upload aborted"
```

Comment:
Fix this so that verify_row considers anything without a proper character to be invalid